### DISCUSSION TITLE
Foundations of Lectical Assessment Course 2023 (first one in 8 years) with Theo Dawson and Zak Stein including introduction to main academic developmental theories

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#435] [General]
https://lecticalive.org/about/fola#gsc.tab=0

https://mailchi.mp/eed37224a13d/a-special-invitation-from-the-lectica-institutea-year-of-developmental-delight-5392832

- What: Course on development and developmental assessment (Lectica model) by (remaining) leading lights of Harvard developmental community
- Cost: $1100
- When: June 2023 for several months

> Announcing Lectica's Year of Developmental Delight!
> 
> We invite you to join Theo Dawson, Zak Stein, and Clint Fuhs for the first live FOLA in over 8 years.
> 
> During our year of developmental delight, a diverse group of hand-picked academics and practitioners will join Theo, Zak, and Clint as discussants. We will engage in rich discussions centered on FOLA Lectures and related readings, including new material covering our research during the period from 2015 to 2023.
> 
> The FOLA is a graduate-level survey course, first delivered in Harvard's Mind, Brain, and Education program. You will primarily be reading material from original sources. But we will not focus exclusively on academics. Our group of discussants will be charged with the task of relating these ideas to real-world educational issues. 
> 
> ...
> 
> The FOLA explores the developmental theory and research that underpin Lectica's learning model and assessments. It includes 11 hour-long recorded lectures ([view sample](https://www.youtube.com/watch?v=Z1Zsu2jqpUU)) that were delivered by Drs. Zak Stein and Theo Dawson during the fall of 2014 at Harvard's Graduate School of Education. Through engagement with these lectures and selected readings, you will explore the pedagogical implications of a wide range of developmental theories, including those of luminaries like Baldwin, Piaget, Erikson, Kohlberg, Loevinger, and Fischer. Throughout the course, you will be asked to consider the real-world pedagogical implications of these theories.

**Comment by baouroux** [comment]
Highly recommended! I've been collaborating with Lectica since 2009, and took the certification early on their launch of the training programmes in 2015.

**Reply by rufuspollock** [reply]
@baouroux could you say more about the assessment system ...

**Reply by baouroux** [reply]
Sure...Lectica have a whole range of assessment designs, for applications in the educational sector (they've done a lot of work in alternative Kindergarten - to - 12 yrs old assessments, that are not 'high-stake' fail/pass assessments, but learning assessments) and in organisational leadership.

Lectica caught my interest back in 2008 in that their assessments are based on setting open-ended and deliberately ambiguous dilemma, that are set in the context of for example in organisational management. This means it's a totally different experience from personality profile tests based on Likert Scales, and goes even further into the 'narrative' of the test-taker than the Sentence Completion Tests originated in the early days of ego/moral developmental assessments.

The participants are asked to write a series of paragraphs, thanks to a series of prompts based on the dilemma description. The assessments then finish with a range of more survey type questions.

Lectica currently principally works in English, although there are solutions for lower accuracy scores in other languages.

The text replies are initially computer scored, thanks to a massive amount of human coding of responses - Lectica has it's own AI here in many ways, as to how language structures can be scored for cognitive complexity. These scores are crucially measuring the structure and clarity of the thinking, rather than the subject-specific expert knowledge.

More accurate final scoring is done by the internal Lectica team, and then the client debriefing by certified coaches.

There are around 5 different test designs, with the flagship coaching assessment being the LDMA (Leadership Decision Making).

Here is my [LDMA report from 2015](https://drive.google.com/file/d/1krFYfJQvkn9A-oVywfv22U4xJecHLc4h/view?usp=sharing) - I scored 11.7 on my developmental level. Out of how much??!! Well, that's an open question as to how far you can go, but here is a [spread of scores from Lectica data](https://drive.google.com/file/d/1VBS_6wmj7mNtPQii9NbrUnoQHOtXbDyB/view?usp=sharing).

Lectica is almost like a forensic analysis of every contruct you present in how you would approach a difficult situation at work. To describe the dimensions of analysis, herewith an extract: 
"_The LDMA focuses on three aspects of decision making:
1. collaborative capacity: the ability to bring together diverse perspectives to develop inclusive, innovative, and effective solutions,
2. contextual thinking: the ability to consider problems in terms of the broader systems and contexts in which they are embedded, and
3. cognitive complexity: the ability to think well about complex issues.
These three broad skills are represented in 8 scales, including (1) developmental level, (2) perspective-taking skill, (3) perspective-seeking
skill, (4) perspective coordination, (5) collaborative capacity, (6) contextual thinking, (7) cognitive coherence, and (8) decision-making process._"

This level of the sub-skills data in the reports, really helps the coaches zoom into a developmental 'next step', during the debriefing.

Clint Fuhs, one of the presenters, is a real authority, he's worked with Lectica for many, many years, and applied Lectica for lots of clients.

**Reply by rufuspollock** [reply]
@baouroux how accurate is this 2005 paper describing what looks like an early version of the process with example questions and answers: https://lifeitself.org/notes/books/dawson-tunik-stein-2004-national-leadership-study

**Comment by baouroux** [comment]
Great to see you've posted a reference to that paper... I guess when you
say 'how accurate' is the paper, you're referring to a comparison to the
current assessment process?

I scanned the dilemma questions from the annex, and that has remained very
much the core approach over the years (e.g. as copied below).

But the paper was based on 2 hours of interviews with a far greater depth
than a typical leadership assessment - such as Lectica's LDMA Leadership
Decision Making Assessment, that takes say an 1hr to complete.

I'm slightly out of date with the very latest developments and packaging by
Lectica, but can certainly invite someone like Jonathan Reams to speak to a
LifeItself group, if you'd like to go deeper. Jonathan and I worked closely
together on sensemaking projects for a couple of years, and he's one of the
most experienced Lectica people in Europe.




-----

Problem-solving interviews Hierarchy:

You notice that your supervisor’s calendar indicates that he is meeting
with one of your employees next week and that the employee scheduled the
meeting. You pride yourself on being approachable and easily accessible to
your employees. In addition, over the last several months, you have been
blindsided on several occasions by your supervisor’s tendency to deal
directly with your employees on his favorite projects.

Standard Probes
- What are the important things to consider in this situation? Why?
- What do you think is an appropriate response to this situation? Why?
- Are there other possible responses to this situation? Can you describe
it/them? Why wouldn’t that be as good as the response you described before?

**Reply by rufuspollock** [reply]
@baouroux wow, could you invite Jonathan to give a talk at the monthly research meeting?

**Reply by baouroux** [reply]
Good news, [Jonathan Reams](https://www.ntnu.edu/employees/jonathan.reams) is willing to present to the Monthly meeting, and the 9th of June session looks to be the best for him.

It turns out that Jonathan is part of the delivery team for the FOLA - Fundamentals of Lectica Assessment programme.

He can present say for 20-30min on the journey so far for Lectica (over the past 20 yrs!), his experience in applying Lectica, and vision of the future applications of developmental assessments. Then open for discussions...

How does that sound? Open of course for any suggestions on the format, and look forward to hearing if that date works for the group.

**Reply by rufuspollock** [reply]
@baouroux this sounds great and perfect for the monthly meeting on 9th June. I think we can suggest he has up to 30m to present though he can obviously present for less time (then questions afterwards).

**Comment by baouroux** [comment]
Sure thing, I'll reach out to Jonathan, and get back to you.

For some background :
https://www.ntnu.edu/employees/jonathan.reams

https://www.jonathanreams.com/work/positions

**Comment by baouroux** [comment]
Sounds good Rufus, I'll pass the timing suggestion on to Jonathan.

Let me know if you'd like any further background information....

And meantime, if ever anyone has any framing type questions within the
umbrella of the post-Piagetian school of research that Lectica is derived
from (for example on the applications), that could be a useful heads-up for
Jonathan...helping him steer his presentation.

Attached is the paper that Jonathan wrote on the Foundations in Lectica
Assessments, from the first running of this course a few years back.

**Reply by rufuspollock** [reply]
@baouroux i think you need to post the paper here on github as attachments don't come through on email.

In terms of topic i think a general intro to the post-piagetian school would be good. you could also point him at https://github.com/orgs/life-itself/discussions/154

**Reply by baouroux** [reply]
[Herewith a link to the paper by Jonathan Reams](https://drive.google.com/file/d/1Rb6xNnjxpXbtpkqeoKHaZNkQLMYgbd3I/view?usp=sharing) wrote on the Foundations in Lectica Assessments, from the first running of this course a few years back.

**Comment by baouroux** [comment]
A couple of links here below from Jonathan in preparing for the 9th of June session (based on him having taken a look at the thread on assessments https://github.com/orgs/life-itself/discussions/154)

- "I can also speak to how the 23 IDGs (Inner Development Goals) were created, since I was involved in that process. I can make a link between the more integrative / dynamic skill approach of Fischer and Mascolo (who I’ve spent a bit of time with now, as did [this](https://www.youtube.com/watch?v=bfBcs_bediQ&t=391s) with) and the IDGs, to help them see it in the context of their focus on the topics you list below."
- "I noticed on the first part of the Github thread mentioned wisdom related measures. You can find a bunch of relevant stuff on this topic in the special issue I did with Roger Walsh on this topic": https://integral-review.org/backissue/vol-11-no-2-mar-2015/

**Reply by laurenwigmore** [reply]
@baouroux Hi John! I hope you're well. Here is the link to share for people to [register for the call on Friday](https://us02web.zoom.us/meeting/register/tZEucuqsqD4qHNy6sJwBY-q56j-3i7sbKE8g#/registration)

If you have any questions or need anything in advance let me know.

**Comment by laurenwigmore** [comment]


**Reply by rufuspollock** [reply]
@laurenwigmore can we track this is the relevant issue https://github.com/life-itself/community/issues/569 rather than this discussion 😄🙏
